scss/style.scss

```
@@ -23,115 +22,107 @@ body {margin: 0;}
   clear: both;
   text-align: center;
 }
-@mixin wraps {
-  text-align:center;
-  margin: 0 130px;
-}
-@mixin img_1 {
-  width: 100%;
-}
 .h1, .h2, .h3, .h4, .h5{
-  @include header_title;
-}
-.h1, .h2, .h3, .h4, .h5, .contact_title {
-  @include header_title2;
+  @include h_title;
 }
-
-.h1, .h2, .h3, .h4, .h5, .contact_title{
-  font-size: 50px;
-}
-
-.headerp, .aboutusp,  .Projectsp, .copyright {
-  @include header_p;
+.headerp, .aboutusp, .Projectsp, .copyright {
+  @include hp;
 }
 .h3, .h4, .h5, .contact_title, .copyright{
   @include title_center
 }
-.range_wraps, .Projects_wraps {
-  @include wraps;
-}
 .range_wraps, .Projects_wraps, .opinions_wraps, .contactp_wraps{
   display: flex;
 }
-
-.opiimg, .contactp_wrap {
-  @include img_1;
-}
-
 .range_wrap, .Projects_wrap, .opinions_wrap, .contactp_wrap {
   display: inline-block;
 }
-.Projects_wrap, .contactp_wrap {
-  margin: 25px;
-}
-.header {
+
+// ここからsection1
+.section1 {
   height: 876px;
   background-image: url("../img/top.png");
   background-repeat: no-repeat;
   background-position:right;
   background-size: 876px 876px;
-  .headerul {
-    @include list;
-    float: right;
-    font-size: 16px;
-    color: #fff;
-    margin: 0 167px;
-    letter-spacing: -0.17px;
-    position: relative;
-    li {
-      display: inline-block;
-      margin: 51px 10px;
-    }
-    img {
-      width: 19pt;
-      height: 18pt;
-      position: absolute;
-      top: 50%;
-      left: 564px;
-      transform: translateY(-50%);
-    }
-  }
-  .logos {
-    position: relative;
-    float: left;
-    .headerlogo1 {
-      width: 85px;
-      height: 84px;
-      margin: 20.1px 20px 20px 121.1px;
+  .section1_box {
+    margin: 0 133px 0 120px;
```

section1_boxは幅1200pxの要素が真ん中にあると考えて下記の方が良いです。
```suggestion
    width: 1200px;
    margin: 0 auto;
```